packages/browser-agent-core/src/features/page-view-timing/instrument/index.js

```diff
@@ -52,6 +54,16 @@ export class Instrument extends FeatureBase {
       documentAddEventListener(e, (...args) => this.captureInteraction(...args));
     })
 
+    /** Interaction-to-Next-Paint */
+    onINP(({name, value}) => {
```

Comment:
Not sure if there is scope for change but as developer who is monitoring web performance we would love to know what element caused the INP issue and in some cases the event type. 

This information is available in the attribution API and wondering if the event target could be included. See docs: https://github.com/GoogleChrome/web-vitals#inp-attribution

e.g.
```
import { onINP } from 'web-vitals/attribution'; // <-- need to change import to get this data

onINP(({name, value, attribution }) => {
   // attribution.eventTarget the element interacted with
  // attribution.eventType I found this less useful but maybe others might find it useful 
})
```

Reply:
While I wholly agree with the reasoning, the application experience team is not quite ready to consume that data just yet and discussions to be had, but it's good to think about it in advance while designing the UI for displaying INP & other timings going forward.

Please consider creating a feature request on [New Relic JIRA board](https://issues.newrelic.com/projects/NEWRELIC) to the `Observability -> Application Experience Browser` team  _or_ through the feedback form in NR1 web UI: 
<img width="365" alt="image" src="https://user-images.githubusercontent.com/28714891/213817673-6555bcdf-5a35-422c-a7ce-2dc80821cee2.png">
This would help the teams drive better feature improvements and user experience for all. Thank you!

Reply:
Thanks @cwli24